phillipphan/bars-backend
# backend for saving a song is complete

Detail:
Backend for saving a song is finished

Testing Instructions:
Postman URL
```
http://localhost:8000/songs
```

Postman Object
```
{
    "songLink": "_jZuz3NEr18",
    "title": "I'm In It",
    "channel": "Kanye West - Topic"
}
```

Make a POST request and see if the object with an ID is returned
